File: app/src/main/java/org/wikiedufoundation/wikiedudashboard/course_detail/uploads/view/CourseUploadsView.java
package org.wikiedufoundation.wikiedudashboard.course_detail.uploads.view;

import org.wikiedufoundation.wikiedudashboard.course_detail.uploads.data.CourseUploadList;
import org.wikiedufoundation.wikiedudashboard.helper.Progressive;
import org.wikiedufoundation.wikiedudashboard.helper.Toaster;

public interface CourseUploadsView extends Progressive, Toaster {
    void setData(CourseUploadList courseUploadList);
}

File: app/src/main/java/org/wikiedufoundation/wikiedudashboard/course_detail/uploads/view/CourseUploadsView.kt
package org.wikiedufoundation.wikiedudashboard.course_detail.uploads.view

import org.wikiedufoundation.wikiedudashboard.course_detail.uploads.data.CourseUploadList
import org.wikiedufoundation.wikiedudashboard.helper.Progressive
import org.wikiedufoundation.wikiedudashboard.helper.Toaster

interface CourseUploadsView : Progressive, Toaster {
    fun setData(courseUploadList: CourseUploadList)
}
